### DISCUSSION TITLE
Exclude lock days from range tooltip count.

### DISCUSSION THREAD

**Discussion by nicogominet** [easepick/easepick#138] [Q&A]
I am excluding weekends using the `LockPlugin`:
```js
filter(date, picked) {
  return [0, 6].includes(date.getDay())
},
```
but the tooltips still counts these days:
![image](https://user-images.githubusercontent.com/5601164/193877343-8971afa2-cac1-40b9-a411-564830f6a856.png)

Is there a way to exclude locks days in tooltip total?

Thanks

**Comment by nathanmedhurstschoolshr** [comment]
I too have struggled with an answer for this, there doesnt seem to be an effective way to remove Sat + Sun from the date picker either which is a shame.

Did you get anywhere with this?

**Reply by nicogominet** [reply]
I haven't had the time to dig any deeper on this yet, but I'm expecting to try to find a solution in coming weeks.
Whether that solution is using maybe `setup` `on('view')` in the lines of [this example](https://jsfiddle.net/y1q9cknt/) or switching to a different package is yet to be seen, sorry.

If you come up with something, please share your findings here, that'd be greatly appreciated.

**Comment by wakirin** [comment]
probably not the best solution, but you can do something like this: https://jsfiddle.net/1dqyet0j/

**Reply by nicogominet** [reply]
Thanks @wakirin that works, but should I be worried about future releases?
How likely are these classes to change?

**Reply by wakirin** [reply]
@nicogominet 
There are no plans to change these classes yet and there are currently no plans for the next release.
Anyway breaking changes will be included in the next major release, so you can use `^1.2` in your package.json file for avoid them.

**Reply by nathanmedhurstschoolshr** [reply]
That works well, maybe its a little out of scope for here, but im looking for a way to make the maxDays work in the same way, so say the maxDays is 14 i only want that to tick down when its over a day thats not a weekend?

**Reply by nathanmedhurstschoolshr** [reply]
> That works well, maybe its a little out of scope for here, but im looking for a way to make the maxDays work in the same way, so say the maxDays is 14 i only want that to tick down when its over a day thats not a weekend?

For anyone looking for something similar, this is what i ended up with, im sure there are better ways of doing it, but hopefully it helps someone: https://jsfiddle.net/vjy5rawh/1/
